js/pickup.js

```diff
@@ -1,5 +1,13 @@
 export const PickupAPI = superclass =>
     class extends superclass {
+        /**
+         * Schedules a pickup for a parcel.
+         *
+         * @param {Object} payload The payload object according to the DHL API standards.
+         * @param {Object} options An object of options to configure the request.
+         * @returns {Object} The HTTP response object.
+         * @see https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section

```

Comment:
Shouldn't this be:
```suggestion
         * @see https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section/operations-tag-pickup
```

Specific to the endpoint you are using here

Reply:
I believe those links point to the same place, happy to change them if you think it's clearer.

Reply:
Hum, taking that into consideration, there is no need to change this 👍